proposals/ARM-support.md

```diff
@@ -0,0 +1,54 @@
+
+# Support for ARM
+
+* [x] Proposed 10/31/2019
+* [ ] Prototype: Not Started
+* [ ] Implementation: Not Started
+* [ ] Specification: Not Started
+
+## Summary
+
+Support the Sensor and Body Tracking SDKs on ARM based boards.
+
+## Feature Scenario
+
+Support for ARM enables our customers to build more stand along solutions utilizing mini- PC platforms. This unlocks more scenarios around robotics, manufacturing and healthcare where an Azure Kinect DK should be more mobile. The feature is also the most asked feature on customer voice.
+
+## Supported Hardware
+
+We are considering to use 2 kinds of Jetson boards for the ARM support since most of our customers prefer these boards:
+
+- Jetson Nano
+    - GPU: Custom 128 CUDA Core GPU
+    - CPU: Quad-Core ARM Cortex-A57 @ 1.43 GHz
+    - 4GB DDR4 RAM
+- Jetson TX2 (to run Body Tracking SDK)
+    - GPU: Custom 256 CUDA Core GPU (Pascal architecture)
+    - CPU: Dual-Core NVIDIA Denver 2 ARMv8 64-bit processor + Quad-Core ARM Cortex-A57 @ 1.43 GHz
+    - 4GB / 8GB DDR4 RAM
+
+## Implementation
+
+There are several large adjustments need to be implemented: //WES: there are 4 repo's that need to be modified, SDK, BT, Depth, and packaging.
+
+- Changes to Sensor SDK, Depth Engine, Body Tracking SDK and packaging repo's
+- Changes to the build process for Sensor SDK, Depth Engine and Body Tracking SDK
+- Changes to the test system for Sensor SDK, Depth Engine and Body Tracking SDK
+- Changes to the release process for both Linux and Windows.
+- Evaluate and make changes if needed to the depth engine code that uses SSE instructions.
+
+We will target releasing ARM binaries for Windows and Linux.
```

Comment:
I think we should pick 1, Windows or Linux. Any data on which one?

Reply:
I'd imagine the majority of people wanting to use Azure Kinect with ARM want to do so running linux on an SBC (such as the jetson)

Reply:
The request we received was for Linux, so we will focus on Linux implementation for this effort.